src/reclaim_python_sdk/utils/session_utils.py

```
@@ -66,4 +66,31 @@ async def update_session(session_id, status):
     except Exception as err:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)